### DISCUSSION TITLE
I am not able to use copilot when I changed my laptop to a new one

### DISCUSSION THREAD

**Discussion by kusumash** [community/community#113153] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

THe error I am getting is 
GitHub Copilot could not connect to server. Extension activation failed: "No access to GitHub Copilot found. You are currently logged in as kusumash."

**Comment by davecheney** [comment, marked as answer]
@kusumash there is no copilot plan associated with this github handle. Were you signed in as @kusumash on your other laptop, or was it a different account?
